.github/workflows/ai-runner-docker.yaml

```diff
@@ -28,14 +28,6 @@ jobs:
       contents: read
     runs-on: ubuntu-latest
     steps:
-      - name: Check out code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          # Check https://github.com/livepeer/go-livepeer/pull/1891
-          # for ref value discussion
-          ref: ${{ github.event.pull_request.head.sha }}
-
```

Comment:
`docker/build-push-action@v6` is configured with `context: "{{defaultContext}}:runner"` meaning that it reads from git directly without looking at the local folder changes.